File: app/src/main/java/com/i3ni/geeknum1/view/RecordView.java
package com.i3ni.geeknum1.view;


import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.view.View;
import android.widget.LinearLayout;
import android.widget.Toast;

import com.i3ni.geeknum1.R;
import com.i3ni.geeknum1.bean.TabBean;

/**
 * Created by Mustang on 19/4/30.
 *
 */

public class RecordView extends BaseTabView {


    public RecordView(Context context) {
        super(context);
    }

    public RecordView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public RecordView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void initView() {
        View view = inflate(getContext(),
                R.layout.view_record, this);
        view.findViewById(R.id.tv_test).setOnClickListener(new OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getContext(),"测试额度",Toast.LENGTH_LONG).show();
            }
        });
    }


}

File: app/src/main/java/com/i3ni/geeknum1/view/RecordView.kt
package com.i3ni.geeknum1.view


import android.content.Context
import android.util.AttributeSet
import android.view.View
import android.widget.Toast

import com.i3ni.geeknum1.R
import kotlinx.android.synthetic.main.view_record.view.*

/**
 * Created by Mustang on 19/4/30.
 *
 */

class RecordView : BaseTabView {


    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {}

    override fun initView() {
          View.inflate(context,
                R.layout.view_record, this)
        tv_test.setOnClickListener { Toast.makeText(context, "测试额度", Toast.LENGTH_LONG).show() }
    }


}

